ksp_plugin_adapter/reference_frame_selector.cs

```diff
@@ -0,0 +1,134 @@
+﻿#define HAS_SURFACE
+#define HAS_BODY_CENTRED_ALIGNED_WITH_PARENT
```

Comment:
No, for UI testing purposes. Once I make it functional (and until it is implemented), those `#define`s will have to go away.